.github/workflows/ci.yml

```diff
@@ -240,3 +240,59 @@ jobs:
         with:
           version: latest
           args: --all-targets --all-features
+
+  version_bump:
+    name: Ensure (MINOR) tag is used when making an API breaking change
+    # Change all of these steps to (MAJOR) after 1.0 release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        if: ${{ !contains(github.event.pull_request.title, '(MINOR)') }}
+      
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        if: ${{ !contains(github.event.pull_request.title, '(MINOR)') }}
+        with:
+          toolchain: nightly
+          override: true
+
+      - name: Get latest existing tag
+        uses: actions-ecosystem/action-get-latest-tag@v1
+        if: ${{ !contains(github.event.pull_request.title, '(MINOR)') }}
+        id: get-latest-tag
+        with:
+          semver_only: true
+
+      - name: Set new version
+        uses: paulhatch/semantic-version@v4.0.2
+        id: set-version
+        if: ${{ !contains(github.event.pull_request.title, '(MINOR)') }}
+        with:
+          tag_prefix: "v"
+          format: "${major}.${minor}.${patch}"
+          major_pattern: "(MAJORXXXX)"
+          minor_pattern: "(MINORXXXX)"
+
+      - name: "Bump crate version (NOTE: Not pushed back to repo!)"
+        if: ${{ !contains(github.event.pull_request.title, '(MINOR)') }}
+        run: |
+          sed -i "s/^version = \"[^\"]*\"$/version = \"$VERSION\"/;" sdk/Cargo.toml
+          git config user.email "nobody@example.com"
```

Comment:
does this email get used in any way?

Reply:
seconding this question

Reply:
Effectively, no, but it's required by git. We're temporarily committing the change to Cargo.toml to bump the version, but not pushing that commit back to the repo (which is why I say "effectively" not used).

Git won't allow you to make a commit, even locally, without a name, e-mail address, and commit message. The e-mail address can be faked, but it has to at least look like an e-mail address.